docs/sources/whatsnew/whats-new-in-v10-2.md

```diff
@@ -67,3 +67,25 @@ The `accessTokenExpirationCheck` feature toggle will be removed in Grafana v10.3
 _Generally available in Grafana Enterprise_
 
 With the current release, we enabled RBAC permission validation (`rbac.permission_validation_enabled` setting) by default. This means that the permissions provided in the request during custom role creation or update are validated against the list of [available permissions and their scopes](https://grafana.com/docs/grafana/<GRAFANA VERSION>/administration/roles-and-permissions/access-control/custom-role-actions-scopes/#action-definitions). If the request contains a permission that is not available or the scope of the permission is not valid, the request is rejected with an error message.
+
+## Correlations
+
+### Correlations editor in Explore
+
+<!-- Kristina Durivage -->
+
+_Available in public preview in Grafana Open Source, Enterprise and Cloud_
+
+Creating correlations has just become easier. Try out our new correlations editor in Explore by selecting the **+ Add > Add correlation** option from the top bar, or from the command palette. The editor shows all possible places where you can place data links and guides you through building and testing target queries. For more information, refer to [the documentation](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/administration/correlations/).
+
+To try out Correlations, enable the `correlations` feature toggle. If you’re using Grafana Cloud and would like to enable this feature, contact Grafana Support.
```

Comment:
Sorry still getting used to the separation of on-prem and Cloud release notes. Forgot this sentence doesn't make sense here. It'll be part of the Cloud version of these entries.
```suggestion
To try out Correlations, enable the `correlations` feature toggle.
```